.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+# This workflow runs when commits are pushed to main or a branch starting with
+# "version-". It checks if any packages require a new release, and if so,
+# creates the crates.io release and git tag.
+name: Release
+on:
+  push:
+    branches:
+      - main
+      - version-*
+permissions:
+  contents: write
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: Swatinem/rust-cache@v2
+      - run: cargo xtask auto-release
+        env:
+          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
```

Comment:
Is this token bound to your account? Or where does it come from?

Reply:
Yes, this token is part of my account, although there's nothing really specific to my account -- any of the three of us with owner access to the uefi crates can create a token at https://crates.io/settings/tokens/new with the `publish-update` scope.

A potential alternative would be to create a service account instead of using a personal account, but then we'd need some way to manage credentials for that, so probably more trouble than it's worth for now.

Reply:
Yeah, I agree.